runtime/src/rad_claims.rs

```diff
@@ -167,7 +170,7 @@ impl<T: Trait> Module<T> {
             h.extend_from_slice(a.as_ref());
         }
 
-        T::Hashing::hash_of(&h).into()
+        T::Hashing::hash(&h).into()
```

Comment:
`T::Hashing::hash_of` scale encodes the payload before hashing. Since these are proofs generated out of a merkle tree, using plain concatenation and hashing, scale encoding breaks the verification.